src/pages/workspaces/workspace/workflows/DataStepContent.js

```diff
@@ -86,69 +94,117 @@ export default class DataStepContent extends Component {
     return h(Modal, {
       title: 'Select Data',
       okButton: h(ButtonPrimary, {
-        tooltip: isChooseSets && _.size(selectedEntities) > 10 && 'Please select 10 or fewer sets',
+        tooltip: Utils.cond(
+          [isChooseSets && _.size(selectedEntities) > 10, () => 'Please select 10 or fewer sets'],
+          [_.isEmpty(selectedEntities), () => 'Please select data']
+        ),
         disabled: !!errors || !this.isValidSelectionModel(),
         onClick: () => onSuccess(newSelectionModel)
       }, 'OK'),
       onDismiss,
       width: 'calc(100% - 2rem)'
     }, [
-      div({ style: { ...Style.elements.sectionHeader, marginBottom: '1rem' } },
-        [`Select ${(isSet ? 'up to 10 ' : '') + rootEntityType}s to process${isSet ? ' in parallel' : ''}`]),
       div({
         style: {
           padding: '1rem 0.5rem', lineHeight: '1.5rem'
         }
       }, [
-        !isSet && h(div, { role: 'radiogroup', 'aria-label': 'Select entities' }, [
-          div([
-            h(RadioButton, {
-              text: `Process all ${count} rows`,
-              name: 'process-rows',
-              checked: isProcessAll,
-              onChange: () => this.setNewSelectionModel({ type: processAll, selectedEntities: {} }),
-              labelStyle: { marginLeft: '0.75rem' }
-            })
-          ]),
-          div([
-            h(RadioButton, {
-              text: 'Choose specific rows to process',
-              name: 'process-rows',
-              checked: isChooseRows,
-              onChange: () => this.setNewSelectionModel({ type: chooseRows, selectedEntities: {} }),
-              labelStyle: { marginLeft: '0.75rem' }
-            })
-          ]),
-          hasSet && div([
-            h(RadioButton, {
-              text: 'Choose existing sets',
-              name: 'process-rows',
-              checked: isProcessMergedSet,
-              onChange: () => this.setNewSelectionModel({ type: processMergedSet, selectedEntities: {} }),
-              labelStyle: { marginLeft: '0.75rem' }
-            })
-          ])
-        ]),
-        !isProcessAll && div({
+        div({ role: 'radiogroup', 'aria-label': 'Select entities' }, [isSet ?
+          h(Fragment, [
+            div([
+              h(RadioButton, {
```

Comment:
For consistency with non-set-launch, should the 'all entities' option be first? Or should we put existing sets first? I feel like using an existing set is the most likely case, especially after you've created it once.

Reply:
Ok, thanks for bringing this up and making us think through it. Now the order is (process all as set, select set components, choose existing set) and behavior is:
- if you have a set already, the default selection is choose existing set
- otherwise the default is process all as set